### DISCUSSION TITLE
File picker

### DISCUSSION THREAD

**Discussion by Barugon** [emilk/egui#1931] [Show and tell]
I know there are native solutions for this but the [Dotrix](https://github.com/lowenware/dotrix) project has a pretty nice solution using egui. I copied the files to a separate library repository and made some modifications. If you're interested, the repository is [here](https://github.com/Barugon/egui_file).

**Comment by emilk** [comment]
Cool! If you publish it on crates.io I'll add it to https://github.com/emilk/egui/issues/1942

**Reply by Barugon** [reply]
Done: https://crates.io/crates/egui_file
